### PR TITLE
Adds missing tests after v8 release

### PR DESCRIPTION
**Pull request by swistach**
### Context
The issue aggregates missing tests for issues mentioned in #7171.

### How has this been tested?
Run `npm run test:e2e`.

### Types of changes
- [x] Tests env improvement

### Related issue(s):
1. #7171

### CODE REVIEW

File: src/plugins/manualRowMove/test/manualRowMove.e2e.js [hunk 1]
```diff
@@ -1387,4 +1387,58 @@ describe('manualRowMove', () => {
...
+      const plugin = hot.getPlugin('manualRowMove');
+
+      plugin.moveRow(0, 4);
+      plugin.persistentStateSave();
+
+      hot.destroy();
+
+      hot = new Handsontable(container, config);
+
+      await sleep(500);
```
Comment (wszymanski) [inline src/plugins/manualRowMove/test/manualRowMove.e2e.js:1411]: Is this sleep really needed? I'm not sure about it.

File: test/helpers/custom-matchers.js [hunk 2]
```diff
@@ -276,6 +276,10 @@ list: ${redColor}${checkedArray.join(', ')}${resetColor} doesn't satisfy the con
      *       a background of the first layer and `H` as the latest layer (most dark).
      * '#' - The hash symbol indicates the currently selected cell without changed background color.
      *
+     * The meaning of the symbol used to describe headers:
+     * '*' - The asterisk symbol indicates selected header.
+     * '-' - The single hyphen symbol indicates highlighted header.
+     *
```
Comment (wszymanski) [inline test/helpers/custom-matchers.js:282]: ❤️

**Review by wszymanski — COMMENTED** [review]
Great job! Just two questions.

File: src/plugins/manualRowMove/test/manualRowMove.e2e.js (hunk 1 above)
Comment (swistach) [reply src/plugins/manualRowMove/test/manualRowMove.e2e.js:1411]: I suggest leaving this `sleep`. Up to v7 columnSorting has been using ObserveChanges - without `sleep` was impossible to catch that problem.
I know ObserveChanges is now deprecated, but remember that scenario should point that we improved its behaviour between v7.4.2 and v8.0.0.

File: src/plugins/hiddenRows/test/selection.e2e.js [hunk 3]
```diff
@@ -272,6 +272,39 @@ describe('HiddenRows', () => {
       `).toBeMatchToSelectionPattern();
     });
 
+    it('should properly render selection if mouse moved over hidden row', () => {
+      spec().$container.css({ margin: '35px' });
```
[the remark this answers is not in the document]
Comment (swistach) [reply src/plugins/hiddenRows/test/selection.e2e.js:276]: We remove `spec().$container` after each `it()`. Lines 8 - 13 in this file.

File: src/plugins/manualRowMove/test/manualRowMove.e2e.js (hunk 1 above)
Comment (wszymanski) [reply src/plugins/manualRowMove/test/manualRowMove.e2e.js:1411]: So maybe this sleep could be much faster? It's quite a small dataset (`5x1`).

Comment (swistach) [reply src/plugins/manualRowMove/test/manualRowMove.e2e.js:1411]: The dataset's size doesn't matter, it's about unpredictable observe changes on CI. I tried 200ms and it worked locally but for CI env was too quick.

**Review by wszymanski — APPROVED** [review]

**Pull request merged**